packages/blue-sdk/src/vault/Vault.ts

```diff
@@ -278,10 +278,10 @@ export class AccrualVault extends Vault implements InputAccrualVault {
   }
 
   /**
-   * The MetaMorpho vault's average APY on its assets, excluding the performance fee.
+   * The MetaMorpho vault's APY on its assets averaged over its markets deposits, excluding the performance fee.
```

Comment:
```suggestion
   * The MetaMorpho vault's APY on its assets averaged over its markets deposits, applying the performance fee.
```

Otherwise it can be misunderstanding ("excluding" generally means "ignoring")